test/types/defaults.ts

```diff
@@ -1,8 +1,8 @@
-import { Chart } from '../../src/types.js';
+import { Chart, TitleOptions, TooltipOptions } from '../../src/types.js';
 
 Chart.defaults.scales.time.time.minUnit = 'day';
 
-Chart.defaults.plugins.title.display = false;
+(Chart.defaults.plugins.title as TitleOptions).display = false;
```

Comment:
This cast is not needed now anymore right?

Reply:
No, viceversa. It is now needed (or we need to use ts-ignore). With previous PR impl, having 2 interfaces and defaults was extending the interface without the possible false value) it wasn't needed.  Now it's needed because TS can not recognize the type:

```console
[types] test/types/defaults.ts(5,30): error TS2339: Property 'display' does not exist on type 'false | TitleOptions'.
[types]   Property 'display' does not exist on type 'false'.
```